src/main/java/io/confluent/kafkarest/SimpleFetcher.java

```diff
@@ -0,0 +1,45 @@
+/**
+ * Copyright 2015 Confluent Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ **/
+package io.confluent.kafkarest;
+
+import io.confluent.kafkarest.simpleconsumerspool.SimpleConsumerPool;
+import kafka.javaapi.FetchRequest;
+import kafka.javaapi.FetchResponse;
+import kafka.javaapi.consumer.SimpleConsumer;
+
+public class SimpleFetcher {
```

Comment:
Up to you, it's a matter of preference. I just figured 1 line of code is simpler than a whole class :)

On Java 1.7, I look forward to that day, but for now, despite 1.7 coming up on EOL, it's still unclear when we'll drop 1.6 support.